README.md

```diff
@@ -1,92 +1,192 @@
 
-## Publishing Packages
+### Publishing 
 
-Two functions are available for publishing applications to hex as packages. The first, `hex publish`, simply packages up what you have as is:
+Two providers are available for publishing packages, `publish` and `cut` 
+
+By default `publish` builds and pushes your package and docs to a repository. See the 
+[docs](https://hexdocs.pm/rebar3_hex/rebar3_hex_publish.html) for more information. 
 
 ``` shell
 $ rebar3 hex publish
 ```
 
-Note that it will display the details of what it is publishing (what files, the version, dependencies) and ask if it should continue, so be sure to read the output carefully and make sure it is publishing what you expected.
-
-Another task, `hex cut` is available to provide some additional functionality around versioning and git tags:
+`cut` is available to provide some additional functionality around versioning and git tags. See the 
+[docs](https://hexdocs.pm/hex/rebar3_hex_cut.html) for more information.
 
 ``` shell
 $ rebar3 hex cut
 ```
 
-This command will ask how to increment (major, minor, patch) the version number based on the last version and can create a git tag and push that tag upstream.
+In either case, both providers will display the details of what is being published 
+(what files, the version, dependencies) and ask if it should continue, so be sure to read the 
+output carefully and make sure it is publishing what you expected.
 
-## Publishing Docs
+### Managing package owners 
+
+Owners can be added, removed, and listed for packages you are an owner of with the `hex owner` command. Packages
+can also be transfered to other registered users on hexpm as well. 
 
 ``` shell
-$ rebar3 hex docs
+$ rebar3 hex owner [add | remove | list | transfer] <package> <email>
 ```
 
-## Adding and Removing Owners of Packages
+See the [docs](https://hexdocs.pm/rebar3_hex/rebar3_hex_owner.html) for more information.
 
-Owners can be added and removed for packages you are an owner of with the `hex owner` command:
+### Retiring packages 
 
-``` shell
-$ rebar3 hex owner [add | remove] <package> <email>
+Packages can be flagged as retired on hexpm via the `retire` provider : 
+
+```
+$ rebar3 hex retire PACKAGE VERSION REASON --message
+```
+
+They can also be unretired in case a mistake was made : 
+
+```
+$ rebar3 hex retire PACKAGE VERSION --unretire 
 ```
+
+See the [docs](https://hexdocs.pm/rebar3_hex/rebar3_hex_retire.html) for more information.
+
+### Organizations
+
+rebar3_hex supports working with organizations via the `organization` provider. 
+
+Simply add your organization to either your global rebar.config (i.e., `~/.config/rebar3/rebar.config` ) or a local
+project rebar.config. 
+
+```erlang
+{hex, [{repos, [#{name => <<"hexpm:your_org">>}]}]}.
+```
+
+Now simply authenticate with with the organization repository. Be sure you have already authenticated with the main
+repository first. 
+
+```
+$ rebar3 hex organization auth hexpm:your_org
```

Comment:
```suggestion
$ rebar3 hex auth    # make sure you're authenticated to the main repo first 
$ rebar3 hex organization auth hexpm:your_org  # authenticate to the org
```

consider something like this, such that the preliminary step is part of the basic instructions.